packages/components/card/src/card.scss

```diff
@@ -0,0 +1,198 @@
+:host {
+  --_aspect-ratio: var(--sl-card-media-aspect-ratio, 4/3);
+  --_orientation-breakpoint: var(--sl-card-orientation-breakpoint, 350px);
+  --_text-width: var(--sl-card-text-width, minmax(50%, 1fr));
+  --_media-width: var(--sl-card-media-width, minmax(0, max-content));
+  --_object-position: var(--sl-card-media-x, 50%) var(--sl-card-media-y, 50%);
+
+  --_box-shadow: var(--sl-elevation-shadow-lg);
+  --_card-background: var(--sl-color-card-background);
+  --_card-border-radius: var(--sl-border-radius-card-default);
+  --_card-border: 1px solid var(--sl-color-card-border);
+  --_content-gap: var(--sl-space-card-gap-content-block) var(--sl-space-card-gap-content-inline);
+  --_content-padding-block: 0 var(--sl-space-card-content-block);
+  --_content-padding-inline: var(--sl-space-card-content-inline);
+  --_font-body: var(--sl-text-card-body);
+  --_font-sub-title: var(--sl-text-card-subtitle);
+  --_font-title: var(--sl-text-card-title);
+  --_media-padding-full: 0;
+  --_media-padding: var(-sl-space-card-media-margins);
+
+  --_media-column: var(--_media-width);
```

Comment:
Maybe we should have `:host` separated here as well like in other components so `host` with variables and separated one with exact usage? :)